### PR TITLE
rules page created

### PR DESCRIPTION
**Pull request by ivinayakg**

### CODE REVIEW

File: screens/rules/index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        </div>
+        <div class="header_link">
+          <a href="/screens/rules/index.html" class="t3 active">All Rules</a>
+        </div>
+        <!-- <div class="header_link">
+            <p class="t3">All Catergories</p>
+          </div>
+          <div class="header_link">
+            <p class="t3">Most Popular</p>
+          </div> -->
```
Comment (tushargoyal812) [inline screens/rules/index.html:45]: remove comments

**Review by tushargoyal812 — COMMENTED** [review]
minor changes and good to merge

File: screens/rules/index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/main.css" />
+    <link rel="stylesheet" href="/screens/rules/index.css" />
+    <script
```
Comment (Mohitkumar33) [inline screens/rules/index.html:9]: I think we should add the JavaScript files before the end of the body tag. I am not sure. If I am wrong then you can correct me.

Comment (ivinayakg) [reply screens/rules/index.html:9]: I'm using async, there so it wont affect.

**Pull request merged**